File: app/src/main/java/bassamalim/hidaya/models/AthkarCategoryDB.java
package bassamalim.hidaya.models;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "athkar_categories")
public class AthkarCategoryDB {

    @PrimaryKey
    @ColumnInfo(name = "category_id")
    private final int category_id;
    @ColumnInfo(name = "category_name")
    private final String category_name;

    public AthkarCategoryDB(int category_id, String category_name) {
        this.category_id = category_id;
        this.category_name = category_name;
    }

    public int getCategory_id() {
        return category_id;
    }

    public String getCategory_name() {
        return category_name;
    }
}

File: app/src/main/java/bassamalim/hidaya/core/data/database/dbs/AthkarCategoryDB.kt
package bassamalim.hidaya.core.data.database.dbs

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "athkar_categories")
data class AthkarCategoryDB(
    @field:ColumnInfo(name = "category_id") @field:PrimaryKey val categoryId: Int,
    @field:ColumnInfo(name = "category_name") val categoryName: String?,
    @field:ColumnInfo(name = "category_name_en") val categoryNameEn: String?
)